sdk/component/conf.go

```diff
@@ -19,10 +19,14 @@ const (
 	errReasonCannotBeNil      = "cannot be nil"
 
 	// Env vars.
-	envVarKeyComponentName   = "SMITHY_COMPONENT_NAME"
-	envVarKeyWorkflowID      = "SMITHY_WORKFLOW_ID"
+	// -- BASE
+	envVarKeyComponentName = "SMITHY_COMPONENT_NAME"
+	envVarKeyInstanceID    = "SMITHY_INSTANCE_ID"
+	// -- LOGGING
 	envVarKeyLoggingLogLevel = "SMITHY_LOG_LEVEL"
-	envVarKeyBackedStoreType = "SMITHY_BACKEND_STORE_TYPE"
+	// -- STORE
+	envVarKeyBackendStoreType     = "SMITHY_BACKEND_STORE_TYPE"
+	envVarKeyBackendLocalStoreDSN = "SMITHY_BACKEND_LOCAL_STORE_DSN"
```

Comment:
I don't see why the `local` substring is needed in these names. the DSN could be anything, either an implementation talking directly to a database or some other implementation using some gateway and the local substring makes them seem like they are only used for the local runner